_includes/intent.md

```diff
@@ -1,15 +1,18 @@
 {% assign autocapture = include.autocapture | default: false %}
+{% assign show-authorization = include.show-authorization | default: true %}
 
 The intent of the payment identifies how and when the charge will be
 effectuated. This determines the type of transaction used during the payment
 process.
+{% unless hide-authorization%}
```

Comment:
You need to replace all usage of `hide-authorization` with `show-authorization` and change`unless` to `if` to reflect the negation.